cmds/sos/sos.go

```diff
@@ -0,0 +1,53 @@
+// Copyright 2018 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package main
+
+import (
+	"fmt"
+	"sync"
+)
+
+var (
+	RWLock   sync.RWMutex
+	Registry = make(map[string]uint)
```

Comment:
Would the new concurrent map fit your use case?

https://golang.org/pkg/sync/#Map

Reply:
I've considered that, but the authors seem to encourage sync.RWMutex + map for regular use cases. 
I didn't use it because we would sacrifice type safety with sync.Map and because in our use cases, keys can be removed, I'm not sure if sync,Map is optimized for our use case.